Tests/GRPCTests/LengthPrefixedMessageReaderTests.swift

```diff
@@ -207,7 +209,7 @@ class LengthPrefixedMessageReaderTests: GRPCTestCase {
 
     // Ensure we can read a message when the rest of the bytes are delivered
     let restOfBytes: [UInt8] = [
-      0x01, // final byte of message
+      0x01  // final byte of message
```

Comment:
The trailing comma should also not be removed. Another bug worth reporting.

Reply:
It's intentional, from https://github.com/apple/swift-format/blob/main/Documentation/Configuration.md:

> multiElementCollectionTrailingCommas (boolean): Determines whether multi-element collection literals should have trailing commas. Defaults to true.


Note __multi-element__. IIRC from digging through issues previously this was to avoid cases where e.g. `let x: [Int]` could be reformatted as:

```
let x: [
  Int,
]
```